jsonnet/telemeter/client/kubernetes/kubernetes.libsonnet

```diff
@@ -0,0 +1,112 @@
+local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
+local credentialsSecret = 'telemeter-client';
+local credentialsVolumeName = 'credentials';
+local credentialsMountPath = '/etc/telemeter';
+local fromCAFile = '/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt';
+local fromTokenFile = '/var/run/secrets/kubernetes.io/serviceaccount/token';
+local metricsPort = 8080;
+
+{
+  _config+:: {
+    namespace: 'default',
+
+    telemeterClient+:: {
+      from: 'https://prometheus-k8s.openshift-monitoring.svc:9091',
```

Comment:
the namespace should be templated here

Reply:
very good call. the current way, users had to update two values, with this change they will only need to change the namespace.